rinex/src/observation/record.rs

```diff
@@ -623,7 +623,7 @@ fn fmt_epoch_v3(
         }
         lines.push('\n');
     }
-    lines
+    lines.trim_end().to_string()
```

Comment:
```suggestion
    lines.truncate(lines.trim_end().len());
    lines
```

(`lines.pop()` is also useful at times)